src/backend/joanie/tests/test_api_course.py

```diff
@@ -208,7 +208,7 @@ def test_api_course_read_detail_anonymous(self):
     def test_api_course_read_detail_authenticated(self):
         """
         Authenticated users should be allowed to retrieve a course
-        with its related order and enrollment bound.
+        with its related order, enrollments and certificates bound.
```

Comment:
I understand there can be only one:
```suggestion
        with its related order, enrollments and certificate bound.
```